patterns/footer-portfolio.php

```diff
@@ -7,13 +7,14 @@
  */
 ?>
 
-<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"7vh","bottom":"7vh"}}}} -->
-<div class="wp-block-group alignwide" style="padding-top:7vh;padding-bottom:7vh"><!-- wp:image {"width":"40px","height":"auto","sizeSlug":"full","linkDestination":"none"} -->
-<figure class="wp-block-image size-full is-resized"><img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/icon-message.webp" alt="" style="width:40px;height:auto"/></figure>
+<!-- wp:group {"align":"wide","layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignwide"><!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"7vh","bottom":"7vh"}}}} -->
+<div class="wp-block-group alignwide" style="padding-top:7vh;padding-bottom:7vh"><!-- wp:image {"width":40,"sizeSlug":"full","linkDestination":"none","className":"is-resized"} -->
```

Comment:
is there a way to do this without two group blocks? how do the other footers handle this?

Reply:
Hmmmm so...

- Writer Footer is _also_ full width - but because it is center aligned, you can't tell.
- main Footer has the Columns block for layout - which has the 'wide' width applied.

Looks like I _might_ be ble to do the same kind of thing with the inner elements of this footer, with the exception of the icon/image as I can't set a width on that block.